app/controllers/katello/products_controller.rb

```diff
@@ -95,5 +89,39 @@ def substitutions
     def task_error(task)
       task.failed_steps.first.action(task.execution_plan).steps.map { |s| s.try(:error).try(:message) }.reject(&:blank?).join(', ')
     end
+
+    def available_kickstart_repos(repos)
```

Comment:
I would rename this to something like  'exclude_rolling_kickstart_repos'.   Not the greatest name, but i was REALLY confused by the name 'available_kickstart_repos'